torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -22,6 +22,31 @@ namespace rpc {
 
 namespace {
 
+void DeleteFutureIValue(FutureIValue* fv) {
```

Comment:
Is there garentee this IValue is always PyObj? Can you add assert of if condition here?